deployment/baremetal/README.md

```diff
@@ -138,8 +138,9 @@ CPX with a builtin Citrix Ingress Controller agent that configures the CPX. CPX
    Official Citrix Ingress Controller docker images is <span style="color:red"> `quay.io/citrix/citrix-k8s-ingress-controller:1.1.1` </span>
 
 4. #### Reachability to the Pod Network:
-    For seamless functioning of services deployed in the Kubernetes cluster, it is essential that Ingress NetScaler device should be able to reach the underlying overlay network over which Pods are running. 
-    `feature-node-watch` knob of Citrix Ingress Controller can be used for automatic route configuration on NetScaler towards the pod network. 
-    Refer [Network Configuration](../../docs/network/staticrouting.md) for further details regarding the same. 
+    For seamless functioning of services deployed in the Kubernetes cluster, it is essential that Ingress NetScaler device should be able to reach the underlying overlay network over which Pods are running.

```

Comment:
Please check Are we using Citrix ADC or Netscler in the documentation. And follow what is most common. 
"Dont use overlay" which will confuse tech people. Just mention, Directly  reaching to  Pod Network from Citrix ADC.